src/test/java/space/obminyashka/items_exchange/end2end/AdvertisementFlowTest.java

```diff
@@ -105,6 +106,17 @@ void findPaginatedAsThumbnails_shouldReturnSpecificAdvertisementTitleDto() throw
                 .andExpect(jsonPath("$.content[0].ownerName").value("admin"));
     }
 
+    @Test
+    @WithMockUser(username = "user")
+    @DataSet("database_init.yml")
```

Comment:
Add @ExpectedDataSet to compare data

Reply:
Don't think that is possible. See: https://github.com/Jack11M/EVO-Exchange-BE-2019/pull/494#discussion_r1117786605